microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,209 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/04/2022
+---
+# What's New in DevTools (Microsoft Edge 99)
+
+The following sections list the announcements from the Microsoft Edge DevTools team.  To try the latest features of DevTools and the Microsoft Edge DevTools extension for Visual Studio Code, read these announcements.  To stay up to date with the latest and greatest features in your developer tools, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) and [follow the Microsoft Edge DevTools team on Twitter](https://twitter.com/EdgeDevTools).
+
+If you are on Windows, Linux, or macOS, consider using the Canary preview channel of Microsoft Edge as your default development browser.  The preview channels give you access to the latest features of Microsoft Edge DevTools.
+
+
+<!-- ====================================================================== -->
+## Securely debug your production code with source maps from Azure Artifacts Symbol Server
+
+<!-- Title: Debug JavaScript with source maps more easily -->
+<!-- Subtitle: Publish your source maps to Azure Artifacts Symbol Server and connect DevTools to it for an easier debugging experience. -->
+
+<!-- todo: sequentially introduce "why source maps" and "why secure source maps" -->
+
+Azure Artifacts Symbol Server now supports storing the source maps that your build process produces when compiling, minifying, and bundling your code.  Instead of hosting source maps on your public server, you can now publish your source maps to the secure Azure Artifacts Symbol Server.  Then, connect DevTools to Symbol Server to have DevTools automatically fetch your source maps. 
+
+By using source maps, you can see and debug your original source code in DevTools, rather than having to work with the compiled, minified, and bundled production code that's returned by your server.  Hosting your source maps on the Azure Artifacts Symbol Server allows you to see and work with your source code securely and privately, instead of placing your source maps on your server and revealing your original code publicly.
+
+To try this feature:
+1. Publish your source maps to Azure Artifacts Symbol Server.
+1. Open DevTools > **Settings** > **Symbol Server** and connect DevTools to Azure Artifacts Symbol Server, by entering your Azure DevOps organization and personal access token.
+
+![The 'Symbol Server' page in DevTools Settings, where you enter your Azure DevOps personal access token.](../../media/2022/03/ado-pat-devtools.png)
+
+See [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md).
```

Comment:
Instead of just linking to this one doc, maybe link to all 3 that are related:
```suggestion
See:
- [Map the processed code to your original source code, for debugging](../../../javascript/source-maps.md),
- [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](../../../javascript/publish-source-maps-to-azure.md).
- [Securely debug original code by using Azure Artifacts symbol server source maps](/javascript/consume-source-maps-from-azure.md).
```